src/apps/activity_assignments/crud/assignments.py

```diff
@@ -57,6 +102,75 @@ async def already_exists(self, schema: ActivityAssigmentSchema) -> bool:
         db_result = await self._execute(select(query))
         return db_result.scalars().first() or False
 
+    async def unassign_many(self, schemas: list[ActivityAssigmentSchema]) -> list[ActivityAssigmentSchema]:
+        """
+        Unassigns multiple assignments by marking them as deleted.
+
+        This method updates the `is_deleted` field for multiple assignment records
+        based on their `id`.
+
+        Parameters:
+        ----------
+        schemas : list[ActivityAssigmentSchema]
+            A list of schemas that need to be updated (unassigned).
+
+        Returns:
+        -------
+        list[ActivityAssigmentSchema]
+            A list of updated assignment schemas.
+        """
+        updated_schemas = []
+        for schema in schemas:
+            updated_schemas.extend(await self._update(lookup="id", value=schema.id, schema=schema))
+        return updated_schemas
+
+    async def get_assignments_by_activity_or_flow_id_and_subject_id(
+        self,
+        search_id: uuid.UUID | None,
+        respondent_subject_id: uuid.UUID,
```

Comment:
we use name of parameters near what their meaning, so I suggest "activity_or_flow_Id" instead of "search_id"

Reply:
Done, changes applied in last commit